rust/src/daemon.rs

```diff
@@ -20,3 +20,49 @@ pub(crate) fn deployment_generate_id(
         deployment.get_deployserial()
     )
 }
+
+pub(crate) fn deployment_populate_variant(
+    mut sysroot: Pin<&mut crate::FFIOstreeSysroot>,
+    mut deployment: Pin<&mut crate::FFIOstreeDeployment>,
+    mut dict: Pin<&mut crate::FFIGVariantDict>,
+) -> CxxResult<()> {
+    let sysroot = &sysroot.gobj_wrap();
+    let deployment = &deployment.gobj_wrap();
+    let dict = dict.gobj_wrap();
+
+    let id = deployment_generate_id(deployment.gobj_rewrap());
+    // First, basic values from ostree
+    dict.insert("id", &id);
+
+    dict.insert("osname", &deployment.get_osname().expect("osname").as_str());
+    dict.insert("checksum", &deployment.get_csum().expect("csum").as_str());
+    dict.insert_value(
+        "serial",
+        &glib::Variant::from(deployment.get_deployserial() as i32),
+    );
+
+    let booted: bool = sysroot
+        .get_booted_deployment()
+        .map(|b| b.equal(deployment))
+        .unwrap_or_default();
+    dict.insert("booted", &booted);
```

Comment:
This slightly changes the behaviour to always inserting the `booted` key instead of only if we're booted inside an OSTree. I vaguely remember having a reason to do that in https://github.com/coreos/rpm-ostree/pull/350, but I can't recall. So... I guess if nothing breaks, let's do it! :)